### ISSUE TITLE
SOCP solves returning status "Unknown"

### ISSUE THREAD

**Issue by chriscoey** [JuliaOpt/Mosek.jl#100]
Mosek.jl is giving us `Unknown` status on some small SOCP problems. Here is the output from one such solve:
```
Problem
  Name                   :
  Objective sense        : min
  Type                   : CONIC (conic optimization problem)
  Constraints            : 80
  Cones                  : 10
  Scalar variables       : 70
  Matrix variables       : 0
  Integer variables      : 0

Optimizer started.
Conic interior-point optimizer started.
Presolve started.
Linear dependency checker started.
Linear dependency checker terminated.
Eliminator started.
Freed constraints in eliminator : 10
Eliminator terminated.
Eliminator - tries                  : 1                 time                   : 0.00
Lin. dep.  - tries                  : 1                 time                   : 0.00
Lin. dep.  - number                 : 0
Presolve terminated. Time: 0.00
Optimizer  - threads                : 1
Optimizer  - solved problem         : the primal
Optimizer  - Constraints            : 10
Optimizer  - Cones                  : 10
Optimizer  - Scalar variables       : 30                conic                  : 30
Optimizer  - Semi-definite variables: 0                 scalarized             : 0
Factor     - setup time             : 0.00              dense det. time        : 0.00
Factor     - ML order time          : 0.00              GP order time          : 0.00
Factor     - nonzeros before factor : 10                after factor           : 10
Factor     - dense dim.             : 0                 flops                  : 9.40e+01
ITE PFEAS    DFEAS    GFEAS    PRSTATUS   POBJ              DOBJ              MU       TIME
0   1.0e+00  9.1e+00  1.0e+00  0.00e+00   0.000000000e+00   0.000000000e+00   1.0e+00  0.00
1   2.1e-01  1.9e+00  1.8e-01  -4.48e-01  2.250776817e+01   3.665115439e+01   2.1e-01  0.00
2   7.7e-02  7.1e-01  6.4e-02  1.14e-01   2.940698748e+02   3.122854389e+02   7.7e-02  0.00
3   1.8e-02  1.7e-01  1.4e-02  1.71e-02   6.857129030e+02   7.090557105e+02   1.8e-02  0.00
4   5.9e-03  5.4e-02  3.7e-03  -1.76e-01  1.116005583e+03   1.155035082e+03   5.9e-03  0.00
5   1.5e-03  1.4e-02  8.1e-04  -2.16e-01  1.867647609e+03   1.923746049e+03   1.5e-03  0.00
6   6.1e-04  5.6e-03  2.5e-04  -3.41e-01  2.595865789e+03   2.688492993e+03   6.1e-04  0.00
7   1.5e-04  1.4e-03  5.4e-05  -2.70e-01  4.271997175e+03   4.401684718e+03   1.5e-04  0.00
8   6.2e-05  5.6e-04  1.8e-05  -3.13e-01  5.863748347e+03   6.058109347e+03   6.2e-05  0.00
9   1.5e-05  1.3e-04  3.5e-06  -2.62e-01  9.510746481e+03   9.788820922e+03   1.5e-05  0.00
10  5.2e-06  4.7e-05  9.5e-07  -3.37e-01  1.358262373e+04   1.405812808e+04   5.2e-06  0.00
11  1.4e-06  1.2e-05  2.2e-07  -2.44e-01  2.137112376e+04   2.199624691e+04   1.4e-06  0.00
12  5.9e-07  5.4e-06  7.7e-08  -3.01e-01  2.838071116e+04   2.929635911e+04   5.9e-07  0.00
13  1.4e-07  1.3e-06  1.5e-08  -2.54e-01  4.561600705e+04   4.691811150e+04   1.4e-07  0.00
14  4.5e-08  4.1e-07  3.7e-09  -3.28e-01  6.727182628e+04   6.956443000e+04   4.5e-08  0.00
15  1.2e-08  1.1e-07  9.0e-10  -2.16e-01  1.037217428e+05   1.066070231e+05   1.2e-08  0.00
16  5.2e-09  4.8e-08  3.2e-10  -3.08e-01  1.370453582e+05   1.414203141e+05   5.2e-09  0.00
17  1.3e-09  1.2e-08  6.4e-11  -2.58e-01  2.199950937e+05   2.262141808e+05   1.3e-09  0.00
18  4.5e-10  8.1e-09  1.8e-11  -3.16e-01  3.116988837e+05   3.217592085e+05   4.5e-10  0.00
19  1.1e-10  1.7e-09  3.8e-12  -2.39e-01  4.921314296e+05   5.056915847e+05   1.1e-10  0.00
20  4.2e-11  5.1e-09  1.1e-12  -3.23e-01  6.843657307e+05   7.069631042e+05   4.2e-11  0.00
21  1.1e-11  6.8e-09  2.4e-13  -2.49e-01  1.088536898e+06   1.119265832e+06   1.1e-11  0.00
22  4.0e-12  6.5e-08  7.5e-14  -3.01e-01  1.498604348e+06   1.545766762e+06   4.0e-12  0.00
23  1.0e-12  8.2e-08  1.6e-14  -2.48e-01  2.373632943e+06   2.439236709e+06   1.0e-12  0.00
24  3.4e-13  2.3e-07  4.1e-15  -3.17e-01  3.410510340e+06   3.521778805e+06   3.4e-13  0.00
25  8.8e-14  2.1e-07  9.1e-16  -2.38e-01  5.355416852e+06   5.504215433e+06   8.8e-14  0.00
26  3.6e-14  1.3e-06  3.1e-16  -3.01e-01  7.197233594e+06   7.421335444e+06   3.6e-14  0.00
27  1.2e-14  4.8e-07  7.5e-17  -2.56e-01  1.088944192e+07   1.119144721e+07   1.1e-14  0.00
28  3.4e-15  2.2e-06  1.6e-17  -2.90e-01  1.755217077e+07   1.807591234e+07   2.6e-15  0.00
29  7.7e-16  4.7e-06  2.7e-18  -2.81e-01  2.727074950e+07   2.809301900e+07   6.8e-16  0.00
30  2.3e-16  1.2e-05  9.9e-19  -2.53e-01  4.039995885e+07   4.154545618e+07   2.1e-16  0.00
31  2.3e-16  1.2e-05  9.9e-19  -2.86e-01  4.039995885e+07   4.154545618e+07   2.1e-16  0.00
Interior-point optimizer terminated. Time: 0.00.

Optimizer terminated. Time: 0.00
```

Would be great to be able to handle a return status properly. I can dump some instances and email if that would help.

**Comment by chriscoey** [comment]
@mlubin @ulfworsoe

**Comment by chriscoey** [comment]
If this is fixed, #73 could perhaps be closed too

**Comment by erling-d-andersen** [comment]
This log indicates that the problem is illposed. You are welcome to dump some instance that we can inspect. You can see at

    https://mosek.com/support/technical-support

how to give us the dumps.

In any case that is what I would always do if you want us at MOSEK to do anything about it.
